W-19544908-crypto-java-dependencies-kt (#2998)
mulesoft/docs-connectors: cryptography/2.0/modules/ROOT/pages/cryptography-module-jce.adoc

Use the Cryptography Module JCE operations to configure these methods.

[NOTE]

> If you are upgrading to Cryptography Module 2.0, note that you need to configure Java dependencies for your project to compile. For details, refer to xref:cryptography-module-upgrade.adoc[]. 

Reviewer: ```suggestion
If you're upgrading to Cryptography Module 2.0, configure the required Java dependencies so your project compiles. For details, refer to xref:cryptography-module-upgrade.adoc[]. 
```

This applies to the below entries.
